plugins/data-management/src/HierarchicalTrackSelectorWidget/model.ts

```diff
@@ -237,17 +338,16 @@ export default function stateTreeFactory(pluginManager: PluginManager) {
         return {
           name: 'Root',
           id: 'Root',
-          children: self.allTracks
-            .map(s => ({
-              name: s.group,
-              id: s.group,
-              children: generateHierarchy({
-                model: self,
-                trackConfs: s.tracks,
-              }),
-            }))
-            // always keep the Tracks entry at idx 0
-            .filter((f, idx) => idx === 0 || !!f.children.length),
```

Comment:
Is this still necessary? I removed it because it was interfering with the ability to toggle Favorites and Recently Used on/off.

This filter is why you couldn't see Recently Used when it was empty

Reply:
can try removing it if you want. note that the comment suggests that it is used to help "keep Tracks entry at idx 0". that assumption i suppose might be broken if recently used and favorites are "above Tracks"

Reply:
What was the original motivation behind forcing it to be 0

Reply:
I think it was just a way to force "Tracks" to display even if it was "empty" but the git blame is not particularly helpful, I didn't make an accurate commit message for this change. I would say can probably delete the idx === 0 part, and then deleting the filter entirely could be ok also if we want to allow showing empty categories (including e.g. completely empty connections)

Reply:
I think it's OK to show empty categories. and important to if we're going to add "functional" categories like these, since they can toggle on and off, and if they're empty and not shown they would be undiscoverable